uptrain/operators/language/llm.py

```diff
@@ -12,6 +12,7 @@
 from loguru import logger
 from tqdm.asyncio import tqdm_asyncio
 from pydantic import BaseModel, Field
```

Comment:
@Dominastorm - can we do lazy_load? Check https://github.com/uptrain-ai/uptrain/blob/9b38b06a7ec06f68afd884665d571986c0d1bfc5/uptrain/operators/embs.py#L25C1-L26C1

Reply:
Done